### ISSUE TITLE
change content type?

### ISSUE THREAD

**Issue by scottdraves** [nfultz/nb2mail#4]
this tool is just what I need, but it doesn't quite work.  the mail file that it produces has
the header:
```
Content-Type: multipart/alternative;
```
but in the result images are not displayed.  just changing it to
```
Content-Type: multipart/mixed;
```
with sed fixed it for me.  

This page indicates it should be mixed (at least the top level):
http://stackoverflow.com/questions/3902455/mail-multipart-alternative-vs-multipart-mixed

seems like this line is the culprit:
https://github.com/nfultz/nb2mail/blob/master/nb2mail/__init__.py#L88

care for a PR?

**Comment by nfultz** [comment]
What mail client are you using? - Gmail has always rendered the images for us.  Happy to make the change since it's so minor though.

**Comment by nfultz** [comment]
I verified that mixed still works on Gmail, so I put it in; fixed in 0.3.
